DEVELOPER_GUIDE.md

```diff
@@ -165,6 +165,10 @@ Run tests :
 ./gradlew clean build test
 ```
 
+## Run Tar file
+```
+tar -xvf opensearch-sdk-1.0.0-SNAPSHOT.tar
```

Comment:
I would suggest adding information on what command to run to build the tarball and also which directory to navigate to in order to find it. We want the details of this guide to be as granular as possible.

Reply:
+1
Also the tar file straight away doesn't run because it needs `extensions.yml` under a specific directory. Lets add all the information.

Reply:
done